### PR TITLE
Run npm audit fix --force and update the tests as necessary

### PR DESCRIPTION
**Pull request by seratch**
###  Summary

To resolve the security vulnerability warnings, I've run `npm audit fix --force` (--force was necessary this time) and fixed the tests that started failing after that.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/tasks-app/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: nodejs/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "jest --no-cache",
     "lint": "eslint . --ext .js",
-    "build": "npx sequelize-cli db:migrate ./database.sqlite3"
+    "build": "npx sequelize-cli db:migrate --to ./database.sqlite3"
```
Comment (seratch) [inline nodejs/package.json:9]: For some reason, the original command does not work for me (The error is `Error: Error parsing url: ./database.sqlite3`). I've confirmed that `npx sequelize-cli db:migrate` is also fine.

File: nodejs/listeners/actions/__tests__/__utils__/action-test-util-funcs.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ const mockActionCallbackInput = (actionPayload) => ({
     },
     chat: {
       update: global.chatUpdateMockFunc,
+      deleteScheduledMessage: global.deleteScheduledMessageMockFunc,
```
Comment (seratch) [inline nodejs/listeners/actions/__tests__/__utils__/action-test-util-funcs.js:13]: I don't know why this mock object had not been necessary but anyway, without this, the test does not pass.

**Review by seratch — COMMENTED** [review]
a few comments for reviewers

**Comment by seratch** [comment]
Hey @colmdoyle, can you take a look at this?

File: nodejs/package.json [hunk 3]
```diff
@@ -25,22 +25,22 @@
...
-    "@slack/bolt": "^3.6.0",
+    "@slack/bolt": "^3.8.1",
     "dotenv": "^10.0.0",
     "eslint-plugin-jest": "^24.4.2",
     "luxon": "^1.28.0",
     "nodemon": "^2.0.12",
     "pluralize": "^8.0.0",
     "sequelize": "^6.6.5",
     "slack-block-builder": "^2.1.1",
-    "sqlite3": "^5.0.2"
```
Comment (colmdoyle) [inline nodejs/package.json:36]: Are we 100% this doesn't break anything? The major semver gives me pause, but I appreciate we barely scratch the surface of DB operations.

Comment (seratch) [reply nodejs/package.json:36]: @colmdoyle I don't know why `npm audit fix` did this but will check later

Comment (seratch) [reply nodejs/package.json:36]: @colmdoyle 
Somehow, sqlite3@5 depends on a very old version range of node-gyp (3.x vs the latest is v8). 
```
$ npm audit
# npm audit report

tar  <=4.4.17
Severity: high
Arbitrary File Creation/Overwrite on Windows via insufficient relative path sanitization - https://github.com/advisories/GHSA-5955-9wpr-37jh
Arbitrary File Creation/Overwrite via insufficient symlink protection due to directory cache poisoning using symbolic links - https://github.com/advisories/GHSA-9r2w-394v-53qc
Arbitrary File Creation/Overwrite due to insufficient absolute path sanitization - https://github.com/advisories/GHSA-3jfq-g458-7qm9
Arbitrary File Creation/Overwrite via insufficient symlink protection due to directory cache poisoning - https://github.com/advisories/GHSA-r628-mhmh-qjhw
fix available via `npm audit fix --force`
Will install sqlite3@4.2.0, which is a breaking change
node_modules/sqlite3/node_modules/tar
  node-gyp  <=3.8.0
  Depends on vulnerable versions of tar
  node_modules/sqlite3/node_modules/node-gyp
    sqlite3  >=5.0.0
    Depends on vulnerable versions of node-gyp
    node_modules/sqlite3

3 high severity vulnerabilities
```

Perhaps, there is something wrong with the sqlite3 npm package distribution in the latest major version. That's why `npm audit fix --force` downgraded the major to the stable v4.2. This should be reasonable for now. What do you think?

Comment (colmdoyle) [reply nodejs/package.json:36]: Yeah the `sqlite3` package has a critical vulnerability they haven't patched.

I just checked out your branch and did a bit of manual testing and everything seems ok, so I say we merge.

**Review by colmdoyle — APPROVED** [review]

**Pull request merged**